### PR TITLE
setup updates

### PR DESCRIPTION
**Pull request by joecheuk**

### CODE REVIEW

File: _docs/_latest/setup/install.md [hunk 1]
```diff
@@ -42,31 +42,37 @@ steps below:
 
 ### Run setup
 
-  1. After you've started Cloud Shell, download Forseti. The installer is
-     included. Getting `2.0-dev-rc1` branch will install Forseti v2.0 Release
-     Candidate 1.
+  1. After you've started Cloud Shell, download Forseti. The installer is included.
```
Comment (mirons-google) [inline _docs/_latest/setup/install.md]: included where? when?

File: _docs/_latest/setup/install.md [hunk 2]
```diff
@@ -42,31 +42,37 @@ steps below:
...
 
      To install both client and server:
      
      ```bash 
-     python setup/installer.py
+     python install/gcp_installer.py
      ```
 
-      To see additional configurations for the setup:
+     To see additional configurations for the setup:
```
Comment (mirons-google) [inline _docs/_latest/setup/install.md]: Please disregard use of a colon before we show something, it's not needed.

Comment (mirons-google) [reply _docs/_latest/setup/install.md]: Please fix this everywhere in this doc, e.g. L90.

File: _docs/_latest/setup/install.md (hunk 1 above)
Comment (mirons-google) [inline _docs/_latest/setup/install.md]: Please put a `---` before the first section.

Comment (joecheuk) [reply _docs/_latest/setup/install.md]: Updated

Comment (joecheuk) [reply _docs/_latest/setup/install.md]: Updated

File: _docs/_latest/setup/install.md (hunk 2 above)
Comment (joecheuk) [reply _docs/_latest/setup/install.md]: Updated

**Pull request merged**